docs/sources/datasources/prometheus.md

```diff
@@ -27,6 +27,23 @@ To access Prometheus settings, hover your mouse over the **Configuration** (gear
 | `Disable metrics lookup`  | Checking this option will disable the metrics chooser and metric/label support in the query field's autocomplete. This helps if you have performance issues with bigger Prometheus instances.           |
 | `Custom Query Parameters` | Add custom parameters to the Prometheus query URL. For example `timeout`, `partial_response`, `dedup`, or `max_source_resolution`. Multiple parameters should be concatenated together with an '&amp;'. |
 
+### Exemplars
+
+> **Note:** This feature is available in Prometheus 2.25+.
+
+The Exemplars configuration allows you to add multiple endpoints to any of your Exemplar label.
+
+You can use this functionality to link to your tracing backend from an Exemplar.
+{{< docs-imagebox img="/img/docs/v74/exemplars-settings.png" class="docs-image--no-shadow" caption="Screenshot of the Exemplars configuration" >}}
+Each Exemplar link config consists of:
+
+- **Label name -** The name of the field in the labels object that should be used.
+- **URL -** If the link is external, then enter the full link URL. You can interpolate the value from the field with `${__value.raw }` macro.
+- **Internal link -** Select if the link is internal or external. In case of internal link, a data source selector allows you to select the target data source. Only tracing data sources are supported.
```

Comment:
This should be in the table above for add data source configuration.

Reply:
I didn't add this to the configuration table to be able to distinguish these configs as these are for exemplars. I've seen that we did this in Loki documentation for derived links. What do you think?

Reply:
@zoltanbedi, when I installed the beta build, and I was adding Prometheus data source, I saw these options. They should be in the table, with a reference to Exemplars content. The content itself should follow the introduction section. I do not think it makes sense to keep the content separate. It can make the content choppy.